WinUIGallery/Helper/NavigationOrientationHelper.cs

```diff
@@ -55,6 +55,9 @@ public static void UpdateTitleBarForElement(bool isLeftMode, UIElement element)
             CoreApplication.GetCurrentView().TitleBar.ExtendViewIntoTitleBar = isLeftMode;
 
             ApplicationViewTitleBar titleBar = ApplicationView.GetForCurrentView().TitleBar;
+#else
+            var window = App.StartupWindow;
```

Comment:
after the titlebar change that just went in, this should be:
 var window = WindowHelper.GetWindowForElement(element);

Reply:
Fixed